src/EntityFramework.Storage/Interfaces/IPersistedGrantDbContext.cs

```diff
@@ -50,6 +51,11 @@ public interface IPersistedGrantDbContext : IDisposable
     /// </value>
     DbSet<ServerSideSession> ServerSideSessions { get; set; }
 
+    /// <summary>
+    /// Gets the database facade
+    /// </summary>
+    DatabaseFacade Database { get; }
```

Comment:
This _is_ a breaking change for any consumers that are not backing their `IPersistedGrantDbContext` with a `DbContext`. I can't imagine someone actually built a fake for this interface, but I suppose not impossible.

Reply:
I've thought of some alternatives, possibly something like:
```
Task BeginTransactionAsync() { return Task.CompletedTask; }
Task CommitTransactionAsync() { return Task.CompletedTask; }
```
But disposing might become a bit tricky (could be done in `CommitTransactionAsync`.

Reply:
Or possibly having a default implementation of `GetAndDeleteAsync` on the `IPersistedGrantDbContext` and then override it with a SELECT and DELETE in one transaction.

```
public interface IPersistedGrantDbContext
{
    async Task<PersistedGrant?> GetAndDeletePersistedGrant(string code)
    {
        var result = await PersistedGrants.SingleOrDefaultAsync(x => x.Key == code);
        await PersistedGrants.Where(x => x.Key == code).ExecuteDeleteAsync();
        return result;
    }
}
```